lib/decimal_arithmetic.ex

```diff
@@ -290,4 +290,11 @@ defmodule DecimalArithmetic do
   def sigil_m(string, []) do
     D.new(string)
   end
+
+  def to_decimal(a) when is_integer(a) do
```

Comment:
Have you left this function public on purpose? I suppose we don't have to expose it as a part of the library's api.

Reply:
No, not on purpose. Private now :)